mangaki/mangaki/utils/knn.py

```diff
@@ -100,13 +80,6 @@ def fit(self, X=None, y=None, whole_dataset=False):
             X = []
         if y is None:
             y = []
-        if whole_dataset:
```

Comment:
I think the `whole_dataset` argument is no longer used? In that case, aren't `None` invalid values for `X` and `y`?